nautobot/virtualization/api/serializers.py

```diff
@@ -67,64 +40,35 @@ class ClusterGroupSerializer(NautobotModelSerializer):
 
     class Meta:
         model = ClusterGroup
-        fields = [
-            "url",
-            "name",
-            "description",
-            "cluster_count",
-        ]
+        fields = "__all__"
 
 
 class ClusterSerializer(NautobotModelSerializer, TaggedModelSerializerMixin):
     url = serializers.HyperlinkedIdentityField(view_name="virtualization-api:cluster-detail")
-    cluster_type = NestedClusterTypeSerializer()
-    cluster_group = NestedClusterGroupSerializer(required=False, allow_null=True)
-    tenant = NestedTenantSerializer(required=False, allow_null=True)
-    location = NestedLocationSerializer(required=False, allow_null=True)
     device_count = serializers.IntegerField(read_only=True)
     virtualmachine_count = serializers.IntegerField(read_only=True)
 
     class Meta:
         model = Cluster
-        fields = [
-            "url",
-            "name",
-            "cluster_type",
-            "cluster_group",
-            "tenant",
-            "location",
-            "comments",
-            "tags",
-            "device_count",
-            "virtualmachine_count",
-        ]
+        fields = "__all__"
 
 
 #
 # Virtual machines
 #
 
 
-class VirtualMachineSerializer(
-    NautobotModelSerializer, TaggedModelSerializerMixin, StatusModelSerializerMixin, RoleModelSerializerMixin
-):
+class VirtualMachineSerializer(NautobotModelSerializer, StatusModelSerializerMixin, TaggedModelSerializerMixin):
     url = serializers.HyperlinkedIdentityField(view_name="virtualization-api:virtualmachine-detail")
-    location = NestedLocationSerializer(read_only=True, required=False, allow_null=True)
-    cluster = NestedClusterSerializer()
-    tenant = NestedTenantSerializer(required=False, allow_null=True)
-    platform = NestedPlatformSerializer(required=False, allow_null=True)
-    primary_ip = NestedIPAddressSerializer(read_only=True)
-    primary_ip4 = NestedIPAddressSerializer(required=False, allow_null=True)
-    primary_ip6 = NestedIPAddressSerializer(required=False, allow_null=True)
-    local_config_context_schema = NestedConfigContextSchemaSerializer(required=False, allow_null=True)
+    primary_ip = IPAddressSerializer(read_only=True)
```

Comment:
Same suggestion here - IMHO `primary_ip` is redundant since we have `primary_ip4` and `primary_ip6` as existing serializer fields.